misc/Kconfig

```diff
@@ -317,8 +317,15 @@ config BOOTLOADER_MCUBOOT
 	help
 	  This option signifies that the target uses MCUboot as a bootloader,
 	  or in other words that the image is to be chain-loaded by MCUboot.
-	  This sets several required options in order for the image generated
-	  to be bootable using the MCUboot open source bootloader.
+	  This sets several required build system and Device Tree options in
+	  order for the image generated to be bootable using the MCUboot open
+	  source bootloader. Currently this includes:
+	  - Setting TEXT_LOAD_OFFSET to a default value that allows space for
```

Comment:
need a blank line before a list and the continuation lines need to be indented:

    source bootloader. Currently this includes:

    - Setting TEXT_LOAD_OFFSET to a default value that allows space for
      the MCUboot image header
    - Activating SW_VECTOR_RELAY on Cortex-M0 targets with no built-in
      vector relocation mechanisms
    - Including subsys/dfu/boot/mcuboot.overlay when building the Device
      Tree in order to place and link the image at the slot0 offset

Reply:
I did that but then the build fails, seems like Kconfig option help text requires consistent alignment

Reply:
you need to shift the bullets to the right I guess, look at this as if it was rst